docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport.
+---
+
+Access to AWS DynamoDB is provided by [**Teleport Application
+Access**](../../application-access/introduction.mdx) for AWS Console and API.
+
+This guide will help you to:
+- Install the Teleport Application Service.
+- Set up the Teleport Application Service to access the AWS Console and API.
+- Connect to your DynamoDB databases through the Teleport Application Service.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![DynamoDB Self-Hosted](../../../img/database-access/guides/dynamodb_selfhosted.png)
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- `aws` (AWS Command Line Interface) installed and added to your system's
```

Comment:
Should "system's" be "shell's"?

Reply:
This is a very tricky one. Most of DB guides use `your system's PATH`. I think you are right most people updates `PATH` in their profile/shell. Though technically you can also update system's PATH. And, for windows, system's PATH is the only option.

I updated the line to
```
- `aws` Command Line Interface (CLI) tool installed in PATH.
```

What do you think?